packages/components/Switch/src/Switch.styling.ts

```diff
@@ -0,0 +1,95 @@
+import { switchName, SwitchTokens, SwitchSlotProps, SwitchProps } from './Switch.types';
+import { UseStylingOptions, Theme, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, shadowStyles } from '@fluentui-react-native/tokens';
+import { defaultSwitchTokens } from './SwitchTokens';
+
+export const switchStates: (keyof SwitchTokens)[] = [
+  'toggleOn',
+  'toggleOff',
+  'before',
+  'after',
+  'above',
+  'hovered',
+  'focused',
+  'pressed',
+  'disabled',
+];
+
+export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, SwitchTokens> = {
+  tokens: [defaultSwitchTokens, switchName],
+  states: switchStates,
+  slotProps: {
+    root: buildProps(
+      (tokens: SwitchTokens) => ({
+        style: {
+          display: 'flex',
```

Comment:
Is there a reason why we put `display: 'flex` everywhere if it's a default display value? I see this not only in your component

Reply:
Personally I just do it out of habit from web development. Not sure why others do it though.

Reply:
I had the same habit :)